README.md

```diff
@@ -8,35 +8,467 @@ The Pinecone Java client lets JVM applications interact with Pinecone services.
 
 ## Installation
 
-*pinecone-client* can be installed from Maven Central for use as a maven dependency in the following ways:
+*pinecone-client* can be installed from [Maven Central](https://mvnrepository.com/repos/central) for use as a maven dependency in the following ways:
```

Comment:
I would link to the top level artifact directly in maven repo: 

`https://mvnrepository.com/artifact/io.pinecone/pinecone-client`